src/js/components/FormField/FormField.js

```diff
@@ -197,6 +197,31 @@ const FormField = forwardRef(
   ) => {
     const theme = useContext(ThemeContext) || defaultProps.theme;
     const formContext = useContext(FormContext);
+
+    // eslint-disable-next-line max-len
+    const isObject =
+      typeof validate === 'object' &&
+      !Array.isArray(validate) &&
+      validate !== null;
+    // eslint-disable-next-line max-len
+    const isMaxAndThresholdValidation = isObject && 'max' in validate && 'threshold' in validate;
+
+    const getMaxAndThresholdValidation = (value) => {
+      const { max, threshold } = validate;
```

Comment:
Can we set a default threshold of "0.5"? This feels like a reasonable default so callers don't have to set it each time themselves. They can override the default if they need it for their use case.

Reply:
Okay

Reply:
![image](https://user-images.githubusercontent.com/23138717/229108972-74326f4a-6ea8-4eab-b598-59e13124622d.png)
Final result